src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/presentation/NavigationButtons.tsx

```diff
@@ -59,13 +59,13 @@ export function NavigationButtons(props: INavigationButtons) {
   const OnClickNext = () => {
     const runPageValidations = !returnToView && triggers && triggers.includes(Triggers.ValidatePage);
     const runAllValidations = returnToView || (triggers && triggers.includes(Triggers.ValidateAllPages));
```

Comment:
line 60&61 could also be updated to `triggers?.includes(...`

Reply:
Agreed. Fixed.